src/static/css/content.less

```diff
@@ -279,6 +276,10 @@ figure {
 
```

Comment:
It's due to the tables. Once we get this on v4 I'd like to update the table to use the layout we're currently using on cfgov-refresh. That said, I think it's beyond the scope of this PR.